runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordGameEventType.java

```diff
@@ -258,11 +296,118 @@
+	REGION_KEBOS_SWAMP("Kebos Swamp", DiscordAreaType.REGIONS, 4664, 4920, 5174, 5175, 5176, 5430, 5431),
+	REGION_KHARAZI_JUNGLE("Kharazi Jungle", DiscordAreaType.REGIONS, 11053, 11309, 11565, 11821),
+	REGION_KHARIDIAN_DESERT("Kharidian Desert", DiscordAreaType.REGIONS, 12844, 12845, 12846, 12847, 12848, 13100, 13101, 13102, 13103, 13104, 13357, 13359, 13360, 13614, 13615, 13616),
+	REGION_KOUREND("Great Kourend", DiscordAreaType.REGIONS, 6201, 6457, 6713),
+	REGION_KOUREND_WOODLAND("Kourend Woodland", DiscordAreaType.REGIONS, 5942, 6197, 6453),
+	REGION_LAW_ALTAR("Law Altar", DiscordAreaType.REGIONS, 9803),
+	REGION_LEGENDS_GUILD("Legends' Guild", DiscordAreaType.REGIONS, 10804),
+	REGION_LIGHTHOUSE("Lighthouse", DiscordAreaType.REGIONS, 10040),
+	REGION_LITHKREN("Lithkren", DiscordAreaType.REGIONS, 14142, 14398),
+	REGION_LUMBRIDGE_SWAMP("Lumbridge Swamp", DiscordAreaType.REGIONS, 12593, 12849),
+	REGION_MAX_ISLAND("Max Island", DiscordAreaType.REGIONS, 11063),
+	REGION_MCGRUBORS_WOOD("McGrubor's Wood", DiscordAreaType.REGIONS, 10550),
+	REGION_MIND_ALTAR("Mind Altar", DiscordAreaType.REGIONS, 11083),
+	REGION_MISTHALIN("Misthalin", DiscordAreaType.REGIONS, 12594, 12595),
+	REGION_MOLCH("Molch", DiscordAreaType.REGIONS, 5177),
+	REGION_MOLCH_ISLAND("Molch Island", DiscordAreaType.REGIONS, 5432),
+	REGION_MORYTANIA("Morytania", DiscordAreaType.REGIONS, 13619, 13620, 13621, 13622, 13876, 13877, 13879, 14133, 14134, 14389, 14390, 14391, 14645, 14647),
+	REGION_MOUNT_QUIDAMORTEM("Mount Quidamortem", DiscordAreaType.REGIONS, 4662, 4663, 4918, 4919),
+	REGION_MUDSKIPPER("Mudskipper Point", DiscordAreaType.REGIONS, 11824),
+	REGION_MYTHS_GUILD("Myths' Guild", DiscordAreaType.REGIONS, 9772),
+	REGION_NATURE_ALTAR("Nature Altar", DiscordAreaType.REGIONS, 9547),
+	REGION_NORTHERN_TUNDRAS("Northern Tundras", DiscordAreaType.REGIONS, 6204, 6205, 6717),
+	REGION_OBSERVATORY("Observatory", DiscordAreaType.REGIONS, 9777),
+	REGION_OTTOS_GROTTO("Otto's Grotto", DiscordAreaType.REGIONS, 10038),
+	REGION_OURANIA_HUNTER("Ourania Hunter Area", DiscordAreaType.REGIONS, 9778),
+	REGION_PIRATES_COVE("Pirates' Cove", DiscordAreaType.REGIONS, 8763),
+	REGION_PISCATORIS_HUNTER_AREA("Piscatoris Hunter Area", DiscordAreaType.REGIONS, 9015, 9016, 9271, 9272, 9528),
+	REGION_POH("Player Owned House", DiscordAreaType.REGIONS, 7513, 7514, 7769, 7770),
+	REGION_POISON_WASTE("Poison Waste", DiscordAreaType.REGIONS, 8752, 9008),
+	REGION_PORT_TYRAS("Port Tyras", DiscordAreaType.REGIONS, 8496),
+	REGION_QUARRY("Quarry", DiscordAreaType.REGIONS, 12589),
+	REGION_RANGING_GUILD("Ranging Guild", DiscordAreaType.REGIONS, 10549),
+	REGION_RATCATCHERS_MANSION("Ratcatchers Mansion", DiscordAreaType.REGIONS, 11343),
+	REGION_RUNE_ESSENCE_MINE("Rune Essence Mine", DiscordAreaType.REGIONS, 11595),
+	REGION_SHIP_YARD("Ship Yard", DiscordAreaType.REGIONS, 11823),
+	REGION_SHIPWRECKS("Shipwrecks", DiscordAreaType.REGIONS, 6486, 6487, 6488, 6489, 6742, 6743, 6744, 6745),
```

Comment:
The enum name (and maybe the visible name also?) should be `REGION_DS2_SHIPWRECKS`